server/src/test/java/org/elasticsearch/action/search/ExpandSearchPhaseTests.java

```diff
@@ -338,7 +340,9 @@ void sendExecuteMultiSearch(MultiSearchRequest request, SearchTask task, ActionL
                 .preference("foobar")
                 .routing("baz");
 
-            SearchHits hits = SearchHits.empty(new TotalHits(1, TotalHits.Relation.EQUAL_TO), 1.0f);
+            SearchHit hit = new SearchHit(1, "ID");
+            hit.setDocumentField("someField", new DocumentField("someField", Collections.singletonList("foo")));
+            SearchHits hits = new SearchHits(new SearchHit[] { hit }, new TotalHits(1, TotalHits.Relation.EQUAL_TO), 1.0F);
```

Comment:
As hits were empty, the test was essentially disabled because we skip if 
``
 (isCollapseRequest() == false || searchHits.getHits().length == 0) 
```